static/app/components/workflowEngine/gridCell/actionCell.tsx

```diff
@@ -1,69 +1,16 @@
-import styled from '@emotion/styled';
-
-import {Flex} from 'sentry/components/container/flex';
-import {Tooltip} from 'sentry/components/core/tooltip';
-import {IconCircledNumber} from 'sentry/components/iconCircledNumber';
 import {EmptyCell} from 'sentry/components/workflowEngine/gridCell/emptyCell';
-import {ActionMetadata} from 'sentry/components/workflowEngine/ui/actionMetadata';
-import {space} from 'sentry/styles/space';
 import type {ActionType} from 'sentry/types/workflowEngine/actions';
+import {AutomationActionSummary} from 'sentry/views/automations/components/automationActionSummary';
 
 type ActionCellProps = {
   actions: ActionType[];
   disabled?: boolean;
 };
 
-export function ActionCell({actions, disabled}: ActionCellProps) {
+export function ActionCell({actions}: ActionCellProps) {
```

Comment:
I originally had the `disabled` prop here so that we wouldn't show the tooltip if this row in the detector list is disabled, will be handling this a different way?

Reply:
Oh I didn't realize that! I can add it back in as `hasTooltip={!disabled}`. But is there a reason  the user wouldn't want the tooltip when the automation is disabled?